lightning/src/ln/channelmanager.rs

```diff
@@ -333,6 +333,15 @@ pub(super) struct ChannelHolder<Signer: Sign> {
 	pub(super) pending_msg_events: Vec<MessageSendEvent>,
 }
 
+/// Events which we process internally, but which cannot be procsesed immediately at the
+/// generation site for some reason. They are handled in timer_chan_freshness_every_min, so may be
+/// processed with quite some time lag.
+enum BackgroundManagerEvent {
```

Comment:
How about `BackgroundEvent`? The manager exists in a separate crate and isn't required.